src/marqo/core/search/recommender.py

```diff
@@ -35,7 +35,7 @@ def recommend(self,
                   reranker: Union[str, Dict] = None,
                   filter: str = None,
                   attributes_to_retrieve: Optional[List[str]] = None,
-                  score_modifiers: Optional[ScoreModifier] = None
+                  score_modifiers: Optional[ScoreModifierLists] = None
```

Comment:
Should it be `ScoreModifierList` (singular)?

Reply:
I think it's because it contains two lists?

Reply:
Yes, it contains to lists, one for multiply, one for add. Just wanted to be explicit